src/ILCompiler/src/Program.cs

```diff
@@ -67,61 +68,37 @@ private void InitializeDefaultOptions()
         // https://github.com/dotnet/corert/issues/568
         private void ParseCommandLine(string[] args)
         {
-            var parser = new CommandLineParser(args);
+            IReadOnlyList<string> explicitInputFiles = Array.Empty<string>();
+            IReadOnlyList<string> implicitInputFiles = Array.Empty<string>();
+            IReadOnlyList<string> referenceFiles = Array.Empty<string>();
```

Comment:
Do we really need both the implicit and explicit input files? I know it is how it was done before your change ... but it does not make it a good idea.